recipes/default.rb

```diff
@@ -17,10 +17,14 @@
 # limitations under the License.
 #
 
-include_recipe 'yum-epel' unless node['platform'] == 'fedora'
+include_recipe 'yum-epel' if platform_family?('rhel')
 
 %w(atomic atomic-testing).each do |repo|
   if node['yum'][repo]['managed']
+    if node['yum'][repo]['baseurl'].nil? and node['yum'][repo]['mirrorlist'].nil?
```

Comment:
here if you can just change the "and" to "&&" to make rubocop happy it will be all good :)
```
recipes/default.rb:24:42: C: Use && instead of and.
    if node['yum'][repo]['baseurl'].nil? and node['yum'][repo]['mirrorlist'].nil?
```